tests/features/ecas-register.feature

```diff
@@ -9,3 +10,40 @@ Feature: Register through OE Authentication
     # Redirected to the Ecas mockup server.
     Then I should see "Create an account"
     And I should see "This page is part of the EU login mock."
+
+  @cleanup:user
+  Scenario: Login user with auto-register enabled
+    Given I am an anonymous user
+    Given CAS users:
+      | Username    | E-mail                            | Password           | First name | Last name | Department    | Organisation |
+      | jb007       | 007@mi6.eu                        | shaken_not_stirred | James      | Bond      | DIGIT.A.3.001 | external     |
+    When I am on the homepage
+    And I click "Log in"
+    # Redirected to the mock server.
+    And I fill in "Username or e-mail address" with "007@mi6.eu"
+    And I fill in "Password" with "shaken_not_stirred"
+    And I press the "Login!" button
+    # Redirected back to Drupal.
+    Then I should see "Your account is currently pending approval by the site administrator."
```

Comment:
Maybe we can use here a more exact assertion: _Then I should see the success message "..."_ (still, I don't remember if is success or warning - to be tested).